spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  before(:all) do
+    User.delete_all
```

Comment:
Yes, I had considered using that gem recently. I will probably create a PR to refactor some specs with it this week.